index.html

```diff
@@ -9,9 +9,14 @@
     <meta name="theme-color" content="__APP_PRIMARY_COLOR__" />
     <meta name="description" content="__APP_DESCRIPTION__" />
     <title>__APP_NAME__</title>
-    <link rel="preconnect" href="https://fonts.googleapis.com">
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=Quicksand:wght@300..700&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
```

Comment:
I'd recommend loading fonts using JS, this is because fonts will be different based on current locale code